### DISCUSSION TITLE
Announcement of CWA 3.0 release date in CWA Twitter and CWA Blog?

### DISCUSSION THREAD

**Discussion by MikeMcC399** [corona-warn-app/cwa-website#3290] [General] [answered]
The Bundesministerium für Gesundheit announced the CWA version 3.0 and its release date via [Twitter](https://twitter.com/BMG_Bund/status/1601146184537583617).

It says the release date is (Wednesday), Jan 18, 2023.

Is it planned to pass on this information also:

- on the dedicated CWA Twitter channel https://twitter.com/coronawarnapp?
- in a blog on this website https://www.coronawarn.app/en/blog/?

**Comment by MikeMcC399** [comment]
So I assume from the lack of any response here that the answer is no additional announcements are planned.

Happy Christmas to the team !

🎄  🎅🏻

**Comment by dsarkar** [comment, marked as answer]
Hi @MikeMcC399, indeed, no additional announcements regarding the release 3.0 are planned currently. 

Merry Christmas!
